spark_log_parser/parsing_models/application_model_v2.py

```diff
@@ -597,8 +596,11 @@ def getAllMetaData(self, appobj):
                 "spark_version": appobj.spark_version,
                 "cloud_platform": appobj.cloud_platform,
                 "cloud_provider": appobj.cloud_provider,
+                "cluster_id": appobj.cluster_id
             },
             "spark_params": appobj.spark_metadata,
+            "existsSQL": self.existsSQL,
+            "existsExecutors": self.existsExecutors
```

Comment:
Moved these fields up into the `getAllMetaData` method so that we just have this dictionary contained in 1 place.